src/Request.php

```diff
@@ -654,6 +664,8 @@ private static function limitTelegramRequests($action, array $data = [])
                 'editMessageReplyMarkup',
                 'setChatTitle',
                 'setChatDescription',
+                'setChatStickerSet',
```

Comment:
@jacklul What exactly are the criteria for having methods in the limiter?
I see that some methods are listed, some not.

Reply:
API docs stated that only methods sending something or editing something in the chats are affected by limits.

Reply:
Ok, so these Sticker Set methods should be limited then 👍